constellations/gemini-admin/src/classes/classRow.js

```diff
@@ -0,0 +1,61 @@
+"use strict";
+require("./classRow.styl");
+import React from "react";
+import moment from "moment";
+import { Link } from "react-router-dom";
+
+export class ClassRow extends React.Component {
+    render() {
+        const row = this.props.row;
+        const unpublished = this.props.unpublished;
```

Comment:
This is especially important for weakly typed languages like Go and Javascript, is to have the variable convey a bit about what value it has.
For boolean variables, I like to use something like `isUnpublished`.